src/components/record-report-page/record-item/index.tsx

```diff
@@ -77,13 +77,12 @@ const RecordItemPure = ({
           <SC.SectionSubTitle>
             {localization.dataProcessorContactDetails}
           </SC.SectionSubTitle>
-          {dataProcessorContactDetails && dataProcessorContactDetails && (
-            <ContactInformation
-              contactDetailsInterface={dataProcessorContactDetails}
-            />
-          )}
+          {dataProcessorContactDetails &&
+            dataProcessorContactDetails &&
+            dataProcessorContactDetails.map(dataProcessor => (
```

Comment:
Two things:

- dataProcessorContactDetails is checked twice
- use ratherdataProcessorContactDetails?.map ...